paddleseg3d/datasets/preprocess_utils/load_image.py

```diff
@@ -46,4 +53,55 @@ def load_series(mhd_path):
     numpyOrigin = np.array(list(reversed(itkimage.GetOrigin())))
     numpySpacing = np.array(list(reversed(itkimage.GetSpacing())))
 
-    return numpyImage, numpyOrigin, numpySpacing
\ No newline at end of file
+    return numpyImage, numpyOrigin, numpySpacing
+
+
+def reverse_axes(image):
+    return np.transpose(image, tuple(reversed(range(image.ndim))))
+
+
+def sitk_read(volume_path, orient=True, split=False, load_type=np.float32):
+    # 1. get scan data
+    # sitk wont load files without orthonormal direction cosines, try/catch next line
+    sitk_f = sitk.ReadImage(volume_path)
+    volume_np = sitk.GetArrayFromImage(sitk_f).astype(load_type)
+    dim = sitk_f.GetDimension()
+
+    # 2. change dimensions to z, y, x
+    # s = volume_np.shape
+    # if len(volume_np.shape) == 3:
+    #     if s[0] == s[1]:
+
+    print("volume_np.shape", volume_np.shape)
+    print(sitk_f.GetDirection())
+    print(dim, sitk_f.GetDepth(), sitk_f.GetHeight(), sitk_f.GetWidth())
+    # 3. correct scan orientation, currently only works with 3d image
```

Comment:
Please write some annotation, and make sure this work.

Reply:
will do, all complex functions would have comment